lib/logstash/plugin_mixins/http_client.rb

```diff
@@ -110,6 +121,19 @@ def client_config
         c[:ssl].merge!(truststore_password: @truststore_password.value)
       end
     end
+
+    if (@keystore)
```

Comment:
@andrewvc Can you remove the unnecessary parentheses in the code? Happen to me all the time when I switch languages often.